lang/java/reef-bridge-java/pom.xml

```diff
@@ -75,6 +75,11 @@ under the License.
             <artifactId>commons-codec</artifactId>
             <version>1.9</version>
         </dependency>
+        <dependency>
+            <groupId>io.github.lukehutch</groupId>
+            <artifactId>fast-classpath-scanner</artifactId>
```

Comment:
This code is under the MIT license. It provides missing java reflection functionality to scan a package for classes as in the three lines of code below. There are a lot of open source packages that people have written to solve this problem. After looking at and trying a number of them this appeared to be the most robust, easiest to use, and well maintained one available. I was very hesitant to add a new dependency but implementation this code ourselves would be a serious commitment of time to solve an already solved problem.

 // Build a list of the message reflection classes.
    final ScanResult scanResult = new FastClasspathScanner("org.apache.reef.bridge.message").scan();
    final List<String> scanNames = scanResult.getNamesOfSubclassesOf(SpecificRecordBase.class);
    final List<Class<?>> messageClasses = scanResult.classNamesToClassRefs(scanNames);